Tests/Marketplace/approved_categories.json

```diff
@@ -2,18 +2,14 @@
     "description": "Approved categories for use in pack_metadata.json and integration yml. To modify this file you will need a CODEOWNER to approve. Please make sure to trigger the sdk build in case you edit this list.",
 	"approved_list": [
 	"Analytics & SIEM",
-	"Authentication",
 	"Case Management",
 	"Cloud Services",
 	"Data Enrichment & Threat Intelligence",
 	"Database",
-	"Deception",
-	"Deception & Breach Simulation",
```

Comment:
Why did you remove this also?

Reply:
I have no memory of removing this not sure how :|